firebase-database/src/androidTest/java/com/google/firebase/database/DataTest.java

```diff
@@ -2740,6 +2744,186 @@ public void onComplete(DatabaseError error, boolean committed, DataSnapshot curr
         < 2000);
   }
 
+  @Test
+  public void testServerIncrementOverwritesExistingData() throws DatabaseException, TimeoutException, InterruptedException {
```

Comment:
This code doesn't seem to be formatted correctly. There is a Gradle command you can run:

`./gradlew :firbase-database:googleJavaFormat`

Reply:
k